### DISCUSSION TITLE
Progetto Esame Ludovica Gatti

### DISCUSSION THREAD

**Discussion by G4tten** [deneb-alpha/Lab_open_source-2023_2024#25] [Projects]
### **Parte principale:**

1. 4 bug su bugzilla
2. Presentazione del repository su Github con la raccolta dei bug
3. Richard Stallman, Linus Torvalds, differenze tra software libero e software open-source

### **Parte facoltativa:**

1. 1 bug demo
2. caricamento del bug demo sul repository tramite terminale

**Comment by deneb-alpha** [comment]
Ciao @G4tten ,
per il punto 3, ti suggerirei anche di mettere a confronto [FSF](https://www.fsf.org/) e [OSI](https://opensource.org/).

Torvalds ha sicuramente avuto un ruolo importante, ma al giorno d'oggi parliamo principalmente di FSF per free software e OSI quando ci si riverisce in odo più generico all'open source. 

Mi raccomando, mi interessa molto anche capire com'è stata la tua esperienza di contribuzione, sentiti libera di presentare anche le difficoltà incontrate o altri feedback da condividere con la comunità di LibreOffice
